chart/templates/iq-server-deployment.yaml

```diff
@@ -218,7 +218,15 @@ spec:
       initContainers:
         - name: {{ .Release.Name }}-set-iq-persistence-ownership
           image: busybox:1.28
-          command: ["sh", "-c", "chown -R 1000:1000 {{ .Values.iq_server.config.clusterDirectory }}"]
+          command:
+            - "sh"
+            - "-c"
+            - >
+              {{- if .Values.customInitCommand -}}
+              {{- .Values.customInitCommand -}}
```

Comment:
suggestion: `customInitCommand`  --> `pvOwenershipOverride` 
(This is only to customize the chown command, not for anything else)

Reply:
Maybe it is better to have this under the `iq_server` property? 79f04f3